Allow for image file names with spaces

Tamakimouto opened the pull request:
Are you delimiting the image file names for a reason?

If not, instead of delimiting them we can just take the whole thing and pass it through an escape so people can just use 

`
"Normal": "File Name.jpg"
`

Tamakimouto commented:
Also I haven't tested to see if it messes up the previous() function.

HyuchiaDiego commented:
Using spaces on file names is a not a good practice, however it should be able to handle it, the `escape()` function is now deprecated so using the `encodeURI()` function would be another way but that would mess up accents in the files as well, so maybe just a `str.replace(" ","%20")` could be used, however it still is a bad practice and file names shouldn't contain spaces. And about delimiting the file names, what do you mean?

Tamakimouto commented:
I agree, using spaces in file names isn't a good idea, but I think alot of people won't know that. After looking at it again, it seems like it doesn't even need to be escaped; it works right when the string split is removed. Auto indented the file for uniformity.

You were delimiting the string by " ", and returning the first part here:
`
var image = characters[parts[1]]["Images"][parts[2]].split(" ")[0];
`
So when someone has a file name as "Bob Smile.png," it just reads it as "Bob." I didn't know if you were doing this for a reason, so I was asking if it would break something if I just took the whole file name.

Also, I found that the CSS to disable image selecting/Dragging wasn't working on my end, just added a bit to remedy that.

HyuchiaDiego commented:
You are right, I think that split is unnecessary and we can just put the whole name without delimiting it, it shouldn't break anything. About the auto indent, sure, that would be nice, make it 4 spaced tabs, I'll change my editor so that the tabs don't get messed up. I'm still thinking about adding gulp and some tasks to this so we can automate all that but I'm not yet sure since that could make contributing even more complicated to some people.

Tamakimouto commented:
Alright that should be everything.
